sift/client.py

```diff
@@ -273,3 +283,7 @@ def is_ok(self):
             return 204 == self.http_status_code
 
         return self.api_status == 0
+
+class SiftRestException(Exception):
```

Comment:
Can we name this `ApiException`. It's in the sift namespace already. No need to "re-Sift" it.